AppCenter/AppCenter/MSACChannelProtocol.h

```diff
@@ -11,6 +11,11 @@
 #else
 #import "MSACEnable.h"
 #endif
+#if __has_include(<AppCenter/MSACNetworkRequestsProtocol.h>)
```

Comment:
You don't need an additional `__has_include` here. Add the header to the existing `#if-#else` above